ui/lib/core/addon/utils/client-count-utils.ts

```diff
@@ -365,3 +370,8 @@ export interface Counts {
   non_entity_clients: number;
   secret_syncs: number;
 }
+
+export interface OnChangeParams {
+  start_time?: number | undefined;
+  end_time?: number | undefined;
+}
```

Comment:
Since this is no longer used other places, I wonder if moving it back to the component makes the most sense 🤔